packages/3d-web-voice-chat/src/chat-ui/components/voice-chat-ui.tsx

```diff
@@ -0,0 +1,114 @@
+import { createRef, forwardRef, useEffect, useRef, useState, MouseEvent } from "react";
+import { flushSync } from "react-dom";
+import { createRoot, Root } from "react-dom/client";
+
+import HourGlass from "../icons/Hourglass.svg";
+import MicrophoneOn from "../icons/MicrophoneOn.svg";
+
+import styles from "./voice-chat-ui.module.css";
+
+type ChatUIInstance = {
+  addMessage: (username: string, message: string) => void;
+};
+
+type ChatUIProps = {
```

Comment:
These types should probably be renamed to be `VoiceChat` to avoid ambiguity with text chat

Reply:
Fixed on [fourth commit](https://github.com/mml-io/3d-web-experience/pull/42/commits/57bab91009fde2a8bc6575939c6b282ade51b035)